riva/proto/riva_asr.proto

```diff
@@ -120,6 +120,7 @@ message EndpointingConfig {
   optional int32 stop_history = 3;
   optional int32 stop_history_eou = 4;
   optional float stop_threshold = 5;
+  optional float stop_eou_threshold = 6;
```

Comment:
can you add help sting for each of the parameter describing range of values and unit ?

Reply:
Thanks!